client/signup/steps/design-type/index.jsx

```diff
@@ -51,6 +51,7 @@ export default React.createClass( {
 					positionInFlow={ this.props.positionInFlow }
 					fallbackHeaderText={ this.translate( 'What would you like your homepage to look like?' ) }
 					fallbackSubHeaderText={ this.translate( 'This will help us figure out what kinds of designs to show you.' ) }
+					subHeaderText={ this.translate( 'First up, what would you like your homepage to look like?' ) }
```

Comment:
Alternate string suggestion: 
* `translate( 'What would you like your homepage to look like?' )`&mdash; translations: **23**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/signup-design-type-subheader)